### ISSUE TITLE
Question about Translation in Nextcloud Registration

### ISSUE THREAD

**Issue by Chuvantinh** [nextcloud/registration#338]
### Question: 
I have added more text in file src/Admin.vue and the form/email.php. It is <?php p($l->t('Captcha')); ?>

But I do not know how to translate this text. I worte the translated text direct on the file of l10n.

I guess I need to have an account on transifex and run this tool in order to have new translations for text. 
I have found one acticle at [nextcloud  document](https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html)
could you tell me about that or is there any documentation ?

**Comment by nickvergessen** [comment]
Well basically it needs to be commited to the master branch of this repository. Otherwise translations are not synced to transifex.

**Comment by Chuvantinh** [comment]
Thank you, I have to ask my teamleader whatever I can make a pull request or not. This is because, I just created a simple text  captcha with 4 numbers for the page `index.php/apps/registration/` . And a config was added in the  setting of backend such as `show captcha`.

Other reason is that I never have made a pull request for any open source on github. Therefore I'm not confident about committing code.
